### PR TITLE
update(.github): enable CI checks for Falco 0.35.1

### PR DESCRIPTION
**Pull request by leogr**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind feature

> /kind bug

/kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area rules

> /area registry

> /area build

> /area documentation

**Proposed rule [maturity level](https://github.com/falcosecurity/rules/blob/main/CONTRIBUTING.md#maturity-levels)**

> Uncomment one (or more) `/area <>` lines (only for PRs that add or modify rules):

> /area maturity-stable

> /area maturity-incubating

> /area maturity-sandbox

> /area maturity-deprecated

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by LucaGuerra — APPROVED** [review]

**Comment by leogr** [comment]
/hold

Let's wait until we complete the Falco 0.36 release (just to avoid any interference). Then, feel free to unblock this.
cc @LucaGuerra

File: .github/FALCO_VERSIONS
Comment (incertum) [inline .github/FALCO_VERSIONS:1]: @leogr shouldn't it now be just 0.36.0?

**Comment by FedeDP** [comment]
Is there anything else needed for this one? @leogr @LucaGuerra

**Pull request merged**
